packages/dd-trace/src/opentelemetry/span.js

```diff
@@ -191,6 +193,16 @@ class Span {
     return this
   }
 
+  addLink (link) {
+    this._ddSpan.addLink(link)
+    return this
+  }
+
+  // context is { traceId, spanId }
+  getLink (context) {
+    return this._ddSpan.getLink(context)
+  }
```

Comment:
I don't think we can add these as they are not part of the OTel Span type spec. Differently behaving functions of the same name could be added later and that would break us.